convert2rhel/checks.py

```diff
@@ -295,6 +296,17 @@ def ensure_compatibility_of_kmods():
         logger.info("Kernel modules are compatible.")
 
 
+def validate_package_manager_transaction():
+    """Validate the package manager transaction is passing the tests."""
+    logger.info("\nValidate the %s transaction", pkgmanager.TYPE)
```

Comment:
```suggestion
    logger.task("\nValidate the %s transaction", pkgmanager.TYPE)
```

Reply:
@r0x0d, sorry, I missed the `\n` which makes the log entry look like this:
```
[10/07/2022 18:57:05] TASK - [
Validate the dnf transaction] ***
```

Reply:
done. Thanks for spotting that!